test/gtest/uct/ib/test_ud.cc

```diff
@@ -918,6 +918,90 @@ UCS_TEST_SKIP_COND_P(test_ud, ctls_loss,
 
 UCT_INSTANTIATE_UD_TEST_CASE(test_ud)
 
+
+class test_ud_peer_failure : public ud_base_test {
+public:
+    test_ud_peer_failure()
+    {
+        m_err_count = 0;
+    }
+
+    void init()
+    {
+        set_config("UD_TIMEOUT?=3s");
```

Comment:
"UD_TIMEOUT=3s"
BTW should it be large with valgrind?

Reply:
I see that other error-handling tests use 3s and it is ok

Reply:
remove the question mark

Reply:
fixed